stdlib/public/Concurrency/GlobalExecutor.cpp

```diff
@@ -155,9 +175,10 @@ TaskExecutorRef _task_executor_getTaskExecutorRef(
 #pragma clang diagnostic pop
 
 SWIFT_CC(swift)
-static bool swift_task_isOnExecutorImpl(HeapObject *executor,
+static bool swift_task_isOnExecutorImpl(HeapObject *executor, // FIXME(Concurrency): this cannot be made crashing (!!!)
```

Comment:
Inversed; this is the one that MUST be made crashing

```suggestion
static bool swift_task_isOnExecutorImpl(HeapObject *executor, // FIXME(Concurrency): make this crash
```